compiler/sem/semexprs.nim

```diff
@@ -266,8 +266,16 @@ proc maybeLiftType(t: var PType, c: PContext, info: TLineInfo) =
   if lifted != nil: t = lifted
 
 proc isOwnedSym(c: PContext; n: PNode): bool =
+  ## checks to see if the `n`ode is the "owned" sym node
   let s = qualifiedLookUp(c, n, {})
-  result = s != nil and sfSystemModule in s.owner.flags and s.name.s == "owned"
+  result =
+    if s.isError:
+      # XXX: move to propagating nkError, skError, and tyError
+      for e in walkErrors(c.config, s.ast):
```

Comment:
Pretty sure local report does a traversal now, if that's the case we can simply drop the loop from all these places.